invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/DeleteCommunityModal.js

```diff
@@ -75,6 +109,10 @@ export class DeleteCommunityModal extends Component {
   openConfirmModal = () => this.setState({ modalOpen: true });
 
   closeConfirmModal = () => {
+    let initialState = this.INITIAL_STATE;
+    let { loading } = this.state;
+    initialState.loading = loading;
```

Comment:
I am not fully understanding this state manipulation here. Why do we need to modify initial state ?

Reply:
Because I want to maintain the current loading status, because the initial state `loading` is `false`. 
The component doesn't get unmounted when we hit `Close` or lose focus of it.